x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/Distribution/index.tsx

```diff
@@ -175,13 +183,14 @@ export const TransactionDistribution: FunctionComponent<Props> = (
         bucketSize={distribution.bucketSize}
         bucketIndex={bucketIndex}
         onClick={(bucket: IChartPoint) => {
-          if (bucket.sample && bucket.y > 0) {
+          if (!isEmpty(bucket.samples) && bucket.y > 0) {
```

Comment:
Can this be simplified to:

```suggestion
          if (!isEmpty(bucket.samples)) {
```

In other words: `bucket.y` should always be larger than 0 if the bucket has samples